www/in-the-wild.html

```diff
@@ -97,6 +129,16 @@ <h4>History</h4>
             <li>e.g. <a href="http://dp.la/exhibitions/exhibits/show/american-aviatrixes/daredevils">Women with Wings: American Aviatrixes, 1920s</a></li>
         </ul>
     </li>
+    <li>
+        <a href="https://repository.dri.ie/">Digital Repository of Ireland</a>
```

Comment:
Let's not do the double `e.g.`. We can keep both of these links but do it more like the "playing soviet" one you did below, something like `Hair Hurling Ball from National Museum of Ireland: "The Inspiring Ireland Project"`.